inference-engine/src/mkldnn_plugin/mkldnn_exec_network.cpp

```diff
@@ -178,9 +182,12 @@ MKLDNNExecNetwork::MKLDNNExecNetwork(const InferenceEngine::ICNNNetwork &network
     }
 
     _graphs = decltype(_graphs){[&] {
+        OV_ITT_TASK_CHAIN(taskChain, MKLDNNPlugin::itt::domains::MKLDNN_LT, "CreateGraph", "cloneNet");
```

Comment:
Are you sure that you OV_ITT_TASK_NEXT was designed for async execution from different threads? From my point of view instrumentation of CreateGraph is enough.

Reply:
Can be removed

Reply:
removed